src/app/app.component.ts

```diff
@@ -153,8 +150,11 @@ export class AppComponent implements OnInit, OnDestroy {
 		// Populate chart
 		this.chartData$ = selectedApiDatas$.pipe(
 			withLatestFrom(selectedDates$),
-			map(([apiDatas, formValues]) => this.getChartData(apiDatas, formValues.start, formValues.end))
+			map(([apiDatas, formValues]) => this.dateService.getChartData(apiDatas, formValues.start, formValues.end))
 		);
+
+		// Testing API call
+		// this.dataService.getGithubRepositoryData('angular/angular-cli').subscribe(console.log);
```

Comment:
leftover commented out code

Reply:
This is on purpose if you want to test the response of the new API. I would leave it there until it is not implemented properly

Reply:
I'd leave testing to unit tests, etc